src/components/List.jsx

```diff
@@ -51,22 +42,30 @@ const List = (props) => {
   };
 
   return (
-    <Box mt="2rem" display="flex" flexDirection="column">
+    <Box mt="2rem" display="flex" flexDirection="column" className="box-list-wrapper">
       <Grid
         container
-        justifyContent="center"
+        justifyContent="flex-start"
+        className="wrapper-lists"
         spacing={{ xs: 1, sm: 1, md: 2 }}
         columns={{ xs: 1, sm: 8, md: 12 }}
       >
         {allList.map((value, index) => (
-          <Grid item xs={2} sm={6} md={6} key={index}>
+          <Grid
```

Comment:
why need map grid?

Reply:
Grid is corrected helper for much items. He allows adaptive change size elements. I have property `item`. `item` can resize item automatically ( `xs={2} sm={6} md={6}` ). `<Box/>` or `<div>`, they need lot of customization than `Grid`.

Reply:
grit is a global container, and the cards are already mapped inside it